Marlin/src/feature/leds/neopixel.cpp

```diff
@@ -50,14 +68,18 @@ void setup_neopixel() {
 
   #if ENABLED(NEOPIXEL_STARTUP_TEST)
     safe_delay(1000);
-    set_neopixel_color(pixels.Color(255, 0, 0, 0));  // red
+    set_neopixel_color_startup(pixels.Color(255, 0, 0, 0));  // red
     safe_delay(1000);
-    set_neopixel_color(pixels.Color(0, 255, 0, 0));  // green
+    set_neopixel_color_startup(pixels.Color(0, 255, 0, 0));  // green
     safe_delay(1000);
-    set_neopixel_color(pixels.Color(0, 0, 255, 0));  // blue
+    set_neopixel_color_startup(pixels.Color(0, 0, 255, 0));  // blue
     safe_delay(1000);
   #endif
 
+  #ifdef BACKGROUND_NEOPIXEL_LED
+    set_neopixel_color_background();
```

Comment:
What does this do when `BACKGROUND_NEOPIXEL_LED` is zero?

Reply:
Oh, I see. I’ll change the description to “index of LED” rather than “number of LED” so fools like me and non-English-speakers won’t interpret it as “quantity of LEDs” that don’t change.